### DISCUSSION TITLE
Undefined constant "Webkul\UVDesk\MailboxBundle\Console\CURLOPT_NOBODY

### DISCUSSION THREAD

**Discussion by yoseabe** [uvdesk/community-skeleton#723] [Q&A]
![tempsnip](https://github.com/uvdesk/community-skeleton/assets/133876586/195657d3-6b2d-4bd4-9b56-b9727cea8ee0)
Can someone help me?
Why is it not working?

**Comment by papnoisanjeev** [comment]
@yoseabe 
Please check if curl is properly installed on your system or not.

**On Windows:** 
Ensure **php_curl.dll** is enabled in your **php.ini** file.

On Ubuntu/Debian:
**sudo apt-get install php-curl**
